packages/features/ee/payments/pages/payment.tsx

```diff
@@ -81,7 +82,7 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
   const { data, booking: _booking, ...restPayment } = rawPayment;
   const payment = {
     ...restPayment,
-    data: data as unknown as PaymentData,
+    data: data as unknown as StripePaymentData,
```

Comment:
There should a better way to type this data, while implementing a new payment app I'm gonna handle this.